Adapter/ShopwareAdapter/ServiceBus/CommandHandler/Product/HandleProductCommandHandler.php

```diff
@@ -239,4 +246,15 @@ private function getVariationResource()
     {
         return Manager::getResource('Variant');
     }
+
+    /**
+     * @param Image $image
+     */
+    private function removeMediaTranslation(Image $image)
+    {
+        $this->entityManager->getConnection()->delete(
```

Comment:
kannst du das in den TranslationHelper verschieben und auch für alle anderen Elemente implementieren?

Reply:
Verschieben okay, die anderen Translations werden meines Wissens nach automatisch gelöscht.

Reply:
ok :)

Reply:
done. Jetzt werden auch nur die **articleimage** Translations gelöscht